src/tribler-core/tribler_core/components/libtorrent/download_manager/download.py

```diff
@@ -277,8 +277,11 @@ def on_save_resume_data_alert(self, alert):
         # Make save_path relative if the torrent is saved in the Tribler state directory
         if self.state_dir and b'save_path' in resume_data:
             save_path = Path(resume_data[b'save_path'].decode('utf8'))
-            if save_path.exists():
-                resume_data[b'save_path'] = str(save_path.normalize_to(self.state_dir))
+            try:
+                if save_path.exists():
+                    resume_data[b'save_path'] = str(save_path.normalize_to(self.state_dir))
+            except PermissionError:
+                pass
```

Comment:
**Questions:** How does the download behave if the save_path is not set? For eg, after the pass on PermissionError. Instead of passing, the exception should at least be logged.